definitions/ext-db2/definition.yml

```diff
@@ -0,0 +1,18 @@
+domain: EXT
+type: DB2
+synthesis:
+  name: database
+  identifier: database
+  encodeIdentifierInGUID: true
+  conditions:
+  - attribute: event_type
+    value: db2
+  tags:
+    database:
```

Comment:
if the database is the name of the entity it doesn't really add anything to have this field as a tag.

Reply:
Shall I remove the database tag?

Reply:
yes please

Reply:
Done